.travis.yml

```diff
@@ -1,37 +1,40 @@
 sudo: false
 language: python
 python:
+- '2.6'
+- '2.7'
+- '3.3'
+- '3.4'
 - '3.5'
+- nightly
+
+cache: pip
 install: pip install -U tox setuptools_scm
 env:
   matrix:
-  - TESTENV=py26-pytest26
```

Comment:
> it gives more correct results from travis

Could you elaborate? I'm curious because most of my projects use this scheme so I'm interested if there's anything wrong with that approach.

Or do you mean visually the jobs on Travis better reflect the Python version running? If that's the case, there's [tox-travis](https://pypi.python.org/pypi/tox-travis), but never used it for non-trivial tox configurations such as what we use here.